graylog2-web-interface/src/views/components/contexts/AutoRefreshProvider.tsx

```diff
@@ -49,6 +49,12 @@ const AutoRefreshProvider = ({ children, onRefresh }: React.PropsWithChildren<{
     };
   }, [refreshConfig?.enabled, refreshConfig?.interval, onRefresh, animationId]);
 
+  useEffect(() => {
```

Comment:
Instead of this second `useEffect`, we should set `defaultRefreshConfig` as the initial value for `refreshConfig` in the `useState` call.